### DISCUSSION TITLE
how can we test whisper model for longer duration? which parameters we can tune?

### DISCUSSION THREAD

**Discussion by akashAD98** [openai/whisper#207] [Q&A]
model is only giving output for few minutes, I want to perform AUDIO to text recognition for more than hour of video, what are the parameters needs to tune?

**Comment by fleek** [comment]
I have used the cli to transcribe full videos before, are you using the cli or you are using it as python library?  If you are using as a python library and you are using the lower level functions (decode/decode task), you have to split up the audio itself into 30 second slices.  However, if you are using the transcribe function, you are just use it because it has the capability to automatically split the audio into 30 second slices, but you have to take care of the output to file function itself.

**Reply by akashAD98** [reply]
im using python -some collab tutorial, which code part I need to do modifications to?

**Reply by FurkanGozukara** [reply]
I use cmd and it is able to transcribe 2 hours speech as well.

Here my tutorial https://youtu.be/msj3wuYf3d8
